webserver.cc

```diff
@@ -1,31 +1,31 @@
-#include "nginx-configparser/config_parser.h"
-#include "server/server.h"
-#include <cstdlib>
-#include <string>
-
+#include "webserver.h"
 #include <boost/asio.hpp>
-
-int main(int argc, char* argv[]) {
-  if (argc != 2) {
-    printf("Usage: ./webserver <path to config file>\n");
-    return 1;
-  }
-
-  NginxConfigParser config_parser;
-  NginxConfig config;
-  
-  if (!config_parser.Parse(argv[1], &config)) {
+bool Webserver::parse_config(const char* file_name){
+  if (!parser_->Parse(file_name, config_)) {
     printf("Invalid config file");
-    return 1;
+    return false;
   }
+  return true;
 
+}
 
-  std::string port_str = "";
-  if (!config.find(port_str)) {
+bool Webserver::get_port_from_config(std::string& s){
```

Comment:
Single user functions like this shouldn't exist. This adds no logic (for testing, as that should be covered by nginx-configparser tests)